### PR TITLE
Even Numbers

### PR DESCRIPTION
**Pull request by Odigene12**
I created a class and testing things out. Hopefully I'm going in the right direction. I tried not to overthink it.

### CODE REVIEW

**Comment by Odigene12** [comment]
@katerebekah

**Comment by katerebekah** [comment]
:+1:

File: Mathmagician/EvenNumber.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+namespace Mathmagician
+{
+    public class EvenNumber
+    {
+        //create a new number (that's supposed to be even)
+        public int usereven { get; set; }
+
+        //create a counter that will add 2 to each number following the "usereven"
+        public int counter = 2;
```
Comment (jcockhren) [inline Mathmagician/EvenNumber.cs]: @Odigene12 use a public property here instead.

File: Mathmagician.Tests/EvenNumberTests.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        [TestMethod]
+        public void CheckIfUserNumberIsEven()
+        {
+            EvenNumber neweven = new EvenNumber();
+
+            neweven.usereven = 7;
+
+            Assert.IsFalse(neweven.usereven % 2 == 0);
+
+        }
```
Comment (jcockhren) [inline Mathmagician.Tests/EvenNumberTests.cs]: @Odigene12 This test begs to be rewritten. How about redoing this by creating a method (in the EvenNumber class) to test whether a number is even.

File: Mathmagician.Tests/EvenNumberTests.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+        [TestMethod]
+        public void CheckIfAdding2Works()
+        {
+            EvenNumber neweven = new EvenNumber();
+
+            neweven.usereven = 4;
+
+            Assert.AreEqual(6, neweven.usereven + neweven.counter);
+        }
```
Comment (jcockhren) [inline Mathmagician.Tests/EvenNumberTests.cs:32]: @Odigene12 This test also needs to be re-written.

**Pull request closed without merging**